### PR TITLE
Fix typo in SHOW_OBSERVED_VARIANT_ARCHIVE config param

### LINKED ISSUE #2781: A custom param missing in prod and stage config file will hide local observations table when we deploy scout web
See this commit: 
https://github.com/Clinical-Genomics/scout/commit/0a283ebc7b74e206de71e732fd6e357ebc17ed48#diff-f57da8aeea4bac4cd8374cdc4c550502cf8768ba15b7b3d6f6ff424b21076a85

Actions:
- Fix syntax of the param
- Introduce changes so local observations will be shown on Scout and Scout stage @CG Stockholm

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix a typo in SHOW_OBSERVED_VARIANT_ARCHIVE config param

**How to test**:
1. Test locally in the 3 scenarios:
- No SHOW_OBSERVED_VARIANT_ARCHIVE param in config file
- SHOW_OBSERVED_VARIANT_ARCHIVE = True in config file 
- SHOW_OBSERVED_VARIANT_ARCHIVE = False in config file

1. Test on clinical-db stage and you should see archived observations.
2. Test locally and try the scenarios above

**Expected outcome**:
Archived observations panel should be shown when SHOW_OBSERVED_VARIANT_ARCHIVE is present and set to True

**Review:**
- [x] code approved by mikaell
- [x] tests executed by mikaell

### CODE REVIEW

**Comment by northwestwitch** [comment]
I might have gotten a bit confused with this one. I have to test how it works also with cancer cases, what for the review @dnil

**Comment by northwestwitch** [comment]
This is ready for review now!

**Comment by dnil** [comment]
Hmm, logo still displays on stage. But, local test shows it can indeed be disabled by setting ACCREDITATION_BADGE to None or commenting it out.

**Comment by northwestwitch** [comment]
> Hmm, logo still displays on stage. But, local test shows it can indeed be disabled by setting ACCREDITATION_BADGE to None or commenting it out.

Debugging on stage now: 🤔 


image

**Comment by northwestwitch** [comment]
@dnil I changed this to match Clinical-Genomics/servers#598

File: scout/server/config.py [hunk 1]
```diff
@@ -66,7 +67,7 @@
 
 # FEATURE FLAGS
 SHOW_CAUSATIVES = True
-SHOW_OBERVED_VARIANT_ARCHIVE = True
```
Comment (mikaell) [inline scout/server/config.py:69]: 👍

**Review by mikaell — APPROVED** [review]
Code looks good. I'll execute the tests and then this can be merged.

**Comment by mikaell** [comment]
Tests work. Tested locally. Feel free to merge.

**Comment by northwestwitch** [comment]
> Tests work. Tested locally. Feel free to merge.

Thanks!

**Pull request merged**